src/zzz_od/operation/enter_game/open_game.py

```
@@ -24,6 +24,10 @@ def open_game(self) -> OperationRoundResult:
         """
         if self.ctx.game_account_config.game_path == '':
             return self.round_fail('未配置游戏路径，请前往 [ 账户管理 ] -> [ 游戏路径 ] 手动设置')
```

Ruff (0.15.12)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)